modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc

```diff
@@ -43,6 +43,14 @@ image::runtime-fabric-3.png[]
 
 After deployment, you can manage the deployed API proxy from Runtime Manager.
 
+== Enable an HTTPS Proxy
+To enable the proxy to support HTTPs requests:
+
+. Download the API gateway and deploy from API Manager using the instructions in xref:api-manager::download-proxy-task.adoc[Downloading an API Proxy].
+. Open the gateway in Anypoint Studio and insert a keystore (SSL cert) using the instructions in xref::mule-runtime::tls-configuration#use-anypoint-studio-to-configure-tls[Use Anypoint Studio to Configure TLS].
+. Export and deploy an app into Runtime Manager using the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
```

Comment:
Export and deploy the gateway using Runtime Manager...

Reply:
Got it - thank you!